fe/fe-core/src/main/java/com/starrocks/sql/ast/UnitBoundary.java

```diff
@@ -0,0 +1,51 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Inc.
+
+package com.starrocks.sql.ast;
+
+import com.starrocks.analysis.Expr;
+import com.starrocks.analysis.LiteralExpr;
+import com.starrocks.sql.common.ErrorType;
+import com.starrocks.sql.common.StarRocksPlannerException;
+import com.starrocks.thrift.TExprNode;
+
+/*
+ * UnitBoundary used to specify time boundary of time_slice:
+ * FLOOR specify START as result time.
+ * CEIL specify END as result time.
+ */
+public class UnitBoundary extends LiteralExpr {
+    private final String description;
```

Comment:
why not use Enum?

Reply:
We use String more natural, and we could tranfer to be directly. this is like UnitIdentifier.